vmdb/spec/support/evm_spec_helper.rb

```diff
@@ -68,12 +68,6 @@ def self.remote_guid_miq_server_zone
     return guid, server, zone
   end
 
-  # FIXXME - rename uses of this method
-  def self.seed_for_miq_queue
-    MiqRegion.seed
```

Comment:
We lose MiqRegion.seed on each of these AFAIK conversions from seed_for_miq_queue -> create_guid_miq_server_zone, can you verify if we need MiqRegion.seed in any of these?

Thanks @kbrock